services/apps/src/apps/containers.clj

```diff
@@ -91,7 +91,9 @@
   (let [tag  (get-tag image-map)
         name (:name image-map)
         url  (:url image-map)]
-    (when-not (image? image-map)
+    (if (image? image-map)
+      (first (select container-images
+                     (where {:name name :tag tag})))
```

Comment:
Yeah, some of the tests were failing when the image map already existed because the function was returning `nil`.